src/sage/rings/polynomial/polynomial_ring.py

```diff
@@ -1401,6 +1405,14 @@ def random_element(self, degree=(-1,2), *args, **kwds):
             sage: while R.random_element(degree=(-1,2), x=-1, y=1) != R.zero():
             ....:     pass
 
+        Note that if the degree range includes `-1` and ``monic`` is set, it is
+        silently ignored, as `0` is not a monic polynomial::
```

Comment:
```suggestion
        Monic polynomials are also chosen among all monic polynomials
        with degree between the given  ``degree`` argument::
```
I don't think you need to tell people that `0` is not a monic polynomial. :) Same comment about uniformity.